hw/ip/otbn/doc/_index.md

```diff
@@ -568,10 +568,16 @@ A fatal alert can only be cleared by resetting OTBN through the `rst_ni` line.
 ### Idle
 
 OTBN exposes a single-bit `idle_o` signal, intended to be used by the clock manager to clock-gate the block when it is not in use.
-This signal is high when OTBN is not running.
+This signal is in the same clock domain as `clk_i`.
+It is high when OTBN is not running.
```

Comment:
The clock may be running. It may be more accurate to say it is not processing a request, or just say it is not busy... or better yet, it is idle :)

Reply:
The sentence says that **OTBN** is not running. Indeed, the clock may or may not be idle.